src/dune_env.mli

```diff
@@ -36,7 +36,10 @@ module Stanza : sig
 
   val decode : t Dune_lang.Decoder.t
 
+  val empty : t
+
   val find : t -> profile:string -> config option
+  val env_vars : t -> profile:string -> Env.t
```

Comment:
To me it's quite confusing that this function doesn't have an option type like `find`.  I think that the caller should turn the `None` to empty, or we should change `find` to work symmetrically.